corehq/apps/export/forms.py

```diff
@@ -431,7 +433,11 @@ def get_multimedia_task_kwargs(self, export, download_id):
         kwargs['export_is_legacy'] = False
         return kwargs
 
-class FilterCaseCouchExportDownloadForm(BaseFilterExportDownloadForm):
+class GenericFilterCaseExportDownloadForm(BaseFilterExportDownloadForm):
```

Comment:
Need this extra class as the couch and ES filters are called with the same constructor